pages/detailed-example-workflow/Step_3_gene_trees

```diff
@@ -0,0 +1,195 @@
+
+            - {gene_name}.trimmed - the trimmed alignment file used for length filtration of each gene
+
+            - {gene_name}.final - the trimmed alignment file used for tree construction by RAxML
+
+            - RAxML_bipartitions.{gene_name.tre} - the final bootstrapped ML tree for each gene
+
+    - a directory `sgt_constructor_out_<M.D.Y>-local` that contains:
+
+        - a directory `trees` that contains: 
+
+            - {gene_name}.trimmed - the trimmed alignment file used for length filtration of each gene
+
+            - {gene_name}.final - the trimmed alignment file used for tree construction by RAxML
+
+            - RAxML_bipartitions.{gene_name}.tre - the final bootstrapped ML tree for each gene
+
+        - metadata.tsv - the file that contains information about taxa already in the database
+
+        - input_metadata.tsv - the input file that contains information about newly added taxa
+
+        - tree_colors.tsv - the file used by `forest.py` in the next step to color taxa by taxonomic affiliation
+
+    - a file sgt_constructor_out_<M.D.Y>.tar.gz - a compressed file of the directory `sgt_constructor_out_<M.D.Y>-local` created to ease the movement of all required
+    data over to a local machine to render the svg files used in the next step. This is often necessary due to the lack of graphics capabilities of headless servers.
+
+\## - These are standard PREQUAL output files for each gene that PhyloFisher has appended
+the corresponding gene name to. See the [`PREQUAL documentation`](https://amoeba.msstate.edu/phylofisher/pdfs/prequal.pdf) for a thorough explanation
+of their contents.
+
+\$$ - These are standard Divvier output files for each gene that PhyloFisher has appended the
+corresponding gene name to. See the [`Divvier documentation`](https://amoeba.msstate.edu/phylofisher/pdfs/divvier.pdf) for a thorough explanation of their
+contents.
+
+\&& - These are standard RAxML output files for each gene that PhyloFisher has appended the
+corresponding gene name to. See the [`RAxML documentation`](https://amoeba.msstate.edu/phylofisher/pdfs/raxml.pdf) for a thorough explanation of their
+contents.
+
+NOTE: For a detailed explanation of the methodology implemented in `sgt_constructor.py` see "Automated Filtering, Alignment, Trimming, and Gene Tree Construction."
+
+NOTE: If `sgt_constructor.py` dies in the middle of a run, simply provide the `sgt_constructor_out_<M.D.Y>` output directory from the previous run to `sgt_constructor.py` via the `-o` flag in addition to the previous command and the script will pick up where it left off.
+
+NOTE: If sgt_constructor is circumnavigated to use alternative parameters for sequence filtering, alignment, and tree reconstruction the following criteria must be met to renter the workflow and downstream steps perform correctly:
+
+1. Trees must have been built using a maximum likelihood program.  Downstream quality control steps are not set up to interpret Bayesian
```

Comment:
The content is good. The only suggestion I have is to tab it over once so that it is obviously under "NOTE: If sgt_constructor is circumnavigated to ..."